app/game/starter_level.rb

```diff
@@ -56,10 +56,18 @@ def tick args
     @player.move_right if args.state.active_input.key_held?(control_mapping[:right])
     @player.climb if args.state.active_input.key_held?(control_mapping[:climb])
     @player.move_up if args.state.active_input.key_held?(control_mapping[:up])
-    @player.move_down if args.state.active_input.key_held?(control_mapping[:down])
-    if args.state.active_input.key_down_or_held?(control_mapping[:jump])
-      @player.jump if args.state.active_input.key_down?(control_mapping[:jump])
-      @player.jump_accelerate if args.state.active_input.key_held?(control_mapping[:jump])
+    if args.state.active_input.key_held?(control_mapping[:down])
+      if args.state.active_input.key_down_or_held?(control_mapping[:jump])
+        # if we are standing on top of a passable floor, pass down through it!
+        $falling_through = true
```

Comment:
Instead of a global variable here, I think I need to set a flag on Actor so that it understands that it is (temporarily!) not colliding with things beneath it if they are a certain type of tile (pass through).

So I think the change is...
- a method on Actor for passing through (and un-passing-through a tick or two later)
- potentially adjusting map OR adjusting Solid to understand what a passable-through solid looks like